zk_prover/src/circuits/tests.rs

```diff
@@ -76,6 +77,34 @@ mod test {
         assert!(full_verifier(&params, &vk, proof, circuit.instances()));
     }
 
+    #[test]
+    fn test_valid_solvency_with_full_prover() {
+        let circuit = SolvencyCircuit::<L, N_ASSETS, N_BYTES>::init_empty();
+
+        // we generate a universal trusted setup of our own for testing
+        let params = generate_setup_params(10);
```

Comment:
Is the K for the Solvency circuit always have a value that is one less than the K for the Inclusion circuit?